src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -63,6 +64,20 @@ protected static String createRepoURI(final String url, final String projectName
         }
     }
 
+    /**
+     * Will for a given list of search criteria return the first found event
+     */
+    protected static String getPreviousEiffelEvent(final String linkedEiffelEventType, final String projectName,
```

Comment:
This method name indicates that it gets an Eiffel event, but it gets and Eiffel event ID. Please update the name to match what id returns.

Reply:
Name is legacy but i have changed it in 23f9e70